GNUmakefile

```diff
@@ -415,7 +424,17 @@ clean:
 	$(MAKE) -C qemu_mode/unsigaction clean
 	$(MAKE) -C qemu_mode/libcompcov clean
 	$(MAKE) -C src/third_party/libradamsa/ clean
-	-rm -rf unicorn_mode/unicornafl
+	rm -rf qemu_mode/qemu-3.1.1
+ifeq "$(IN_REPO)" "1"
+	$(MAKE) -C unicorn_mode/unicornafl clean || true
```

Comment:
Should probably also `$(MAKE) -C qemu_mode/qemu clean || true`? cc @andreafioraldi

Reply:
no as the patches are reapplied when the build script is called. if qemu has to be rebuild then because we changed the patches, so a make clean does not help